### DISCUSSION TITLE
extend XiaomiCloudConnector

### DISCUSSION THREAD

**Discussion by vyaslav** [PiotrMachowski/Xiaomi-cloud-tokens-extractor#107] [Ideas]
would XiaomiCloudConnector work also with mihealth?
login seems to work, but subsequent calls to hlth.io.mi.com with generated nonce, hash and signature return 401.
If it is compatible with mihealth API, it would be great to extend it, let say to extract runs/health data etc.

**Comment by PiotrMachowski** [comment]
I have no idea, I don't use Mi Health
